src/modules/aws.rs

```diff
@@ -21,7 +78,13 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     module.get_prefix().set_value(AWS_PREFIX);
 
     module.create_segment("symbol", &config.symbol);
-    module.create_segment("profile", &config.profile.with_value(&aws_profile));
+    module.create_segment(
+        "region",
+        &SegmentConfig {
+            value: &aws_region,
+            style: None,
+        },
+    );
```

Comment:
Would it be better to have separate segments for the `profile` and the `region`?